packages/gateway-client/copy.sh

```diff
@@ -3,6 +3,6 @@
 cd /usr/src
 mv ./volumes/gateway_client/assets/libp2p.* .
 rm -rf ./volumes/gateway_client/*
-cp -rf ./athena/dist/packages/gateway-client/. ./volumes/gateway_client
-mv libp2p.* ./volumes/gateway_client/assets/
+cp -rf ./athena/packages/gateway-client/. ./volumes/gateway_client
```

Comment:
Why are we moving the source code instead of the build artifacts?

Reply:
the changes from the multistage dockerfile build changed the path.